File: python/warmup2/array123.py
def array123(nums):
  is123 = False
  for i in range(1, len(nums) - 1):
    if(nums[i-1] == 1 and nums[i] == 2 and nums[i + 1] == 3):
      is123 = True
  
  return is123

File: java_code/warmup2/array123.java
package java_code.warmup2;

public class array123 {

    public boolean array123(int[] nums) {
        if (nums.length < 3)
            return false;

        for (int i = 1; i < nums.length - 1; i++) {
            if (nums[i - 1] == 1 && nums[i] == 2 && nums[i + 1] == 3) {
                return true;
            }
        }

        return false;

    }

}